news/2022-08-04-community-mentorship-program-fall-2022-signups-now-open.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Community Mentorship Program Fall 2022 Signups Now Open"
+date: 2022-08-04 14:00:00 +0000
+---
+
+The 22nd cycle of the Community Mentorship program has arrived! Snatch the opportunity to delve deep into the world of mapping, modding, or storyboarding.
+
+![](/wiki/shared/news/banners/community-mentorship-program.jpg)
+
+The Community Mentorship program serves as a place for upcoming mappers, modders, and storyboarders, to develop their skills under the guidance of more experienced members in the community. If you've ever found yourself wanting to expand your current knowledge this might just be the opportunity you need.
```

Comment:
```suggestion
The Community Mentorship program serves as a place for upcoming mappers, modders, and storyboarders, to develop their skills under the guidance of more experienced members of the community. If you've ever found yourself wanting to expand your current knowledge, this might just be the opportunity you need.
```